pkg/webhook/plugins/requirenodewithfuse/require_node_with_fuse.go

```diff
@@ -58,14 +58,21 @@ func (p *RequireNodeWithFuse) Mutate(pod *corev1.Pod, runtimeInfos map[string]ba
 	requiredSchedulingTerms := []corev1.NodeSelectorTerm{}
 
 	for _, runtime := range runtimeInfos {
-		term, err := getRequiredSchedulingTerm(runtime)
-		if err != nil {
+
+		if runtime == nil {
 			return true, fmt.Errorf("should stop mutating pod %s in namespace %s due to %v",
 				pod.Name,
 				pod.Namespace,
-				err)
+				fmt.Errorf("RuntimeInfo is nil"))
+		}
+
+		// dataset used as hcfs, no need to require node with fuse
+		if runtime.IsHcfsUsed() {
```

Comment:
I think whether using hcfs can be determined in pod label instead of runtimeInfo.